sw/device/tests/sim_dv/spi_passthrough_test.c

```diff
@@ -26,11 +28,18 @@ OTTF_DEFINE_TEST_CONFIG();
 // environment.
 const volatile uint32_t kFilteredCommands;
 
+// Whether to upload write commands and have software relay them.
+const volatile uint8_t kUploadWriteCommands;
```

Comment:
Actually, we should not do this. `bool` does not have a defined size in the C standard, and we're writing values to memory regions that interface outside our specific compiler. Use the fixed-size types only here.

Reply:
I see. thanks for the explanation.

> (I also don't want static because the symbol should show up outside this file's scope.)

is it only used in the current file?

Reply:
So it's not the case for our current build and tools, but in some cases, `static` can cause symbols to not make it to the ELF's symbol table. If that happened, our backdoor mechanism that does the lookup wouldn't work.

So here I'm very purposefully ensuring the symbol is global, and it should not disappear once the compiler is done with this compilation unit.